packages/now-next/src/index.ts

```diff
@@ -355,10 +355,11 @@ export const build = async ({
   }
 
   const exportIntent = await getExportIntent(entryPath);
-  if (exportIntent) {
-    const { trailingSlash } = exportIntent;
+  const userExport = await getExportStatus(entryPath);
+
+  if (exportIntent || userExport) {
+    const { trailingSlash = true } = exportIntent || {};
```

Comment:
Trailing slash now defaults to false in Next 9+.

Reply:
Updated